control_panel_api/authentication.py

```diff
@@ -48,5 +48,8 @@ def authenticate(self, request):
                 email=decoded.get(settings.OIDC_FIELD_EMAIL),
                 name=decoded.get(settings.OIDC_FIELD_NAME),
             )
+            user.save()
```

Comment:
django-newbie question: Do you need the call to `save()` if you already called the `create()`?

Reply:
apparently yes

Reply:
Yeah, just tested in a console, `create()` will save to the DB so `save()` is redundant.